openwisp_radius/tests/test_celery.py

```diff
@@ -0,0 +1,79 @@
+from celery import Celery
```

Comment:
in the other modules we use "test_tasks", can you follow this style please?
https://github.com/openwisp/openwisp-firmware-upgrader/blob/master/openwisp_firmware_upgrader/tests/test_tasks.py